_integration-tests/go.mod

```diff
@@ -5,7 +5,6 @@ go 1.21
 replace github.com/datadog/orchestrion => ../
 
 require (
-	github.com/datadog/orchestrion v0.0.0-00010101000000-000000000000
```

Comment:
It's weird this would disappear?

Reply:
No idea why. I did run `go mod tidy`. Does it happen in your local branch too?

Reply:
Solved. I didn't run it again in `_integration_tests`.